packages/mendel-config/src/bundle-config.js

```diff
@@ -7,6 +7,19 @@ function BundleConfig(id, options, config) {
     this.generator = options.generator || 'default';
 
     const {outdir} = config.baseConfig;
+    const supportedOptions = [
+        'entries',
+        'external',
+        'from',
+        'generator',
+        'manifest',
+        'outfile',
+        'outlet',
+        'require',
+        'runtime',
+    ];
```

Comment:
This warning is for extra props defined in each `bundle`. `Object.keys(defaultConfig)` gives only top level keys of `.mendelrc`